scripts/pylint_custom_plugin/tests/test_pylint_custom_plugins.py

```diff
@@ -2677,3 +2677,72 @@ def test_disable_pylint_alias(self):
 
         with self.assertNoMessages():
             self.checker.visit_module(node)
+
+class TestCheckApiVersion(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = checker.CheckApiVersion
+
+    def test_bad_api_version(self):
+        class_node = astroid.extract_node(
+            """
+            class SomeClient(object):
+                '''
+                   : param str something
```

Comment:
we should use proper docstring syntax in the tests.

```suggestion
                   :param str something:
```